### ISSUE TITLE
fsearthtiles

### ISSUE THREAD

**Issue by MBL820** [stackTom/FSEarthTiles#21]
il est possible d'ajouter une grille et un serveur maps de qualité supérieure?

**Comment by stackTom** [comment]
Oui, jetez un œil aux exemples dans le dossier Providers. Ou dans FSEarthtiles.ini également. Quel serveur souhaitez-vous ajouter ? Je peux vous montrer comment.
